modules/document_repository/ajax/documentEditUpload.php

```diff
@@ -25,15 +25,22 @@
 require_once "NDB_Client.class.inc";
 require_once "NDB_Config.class.inc";
 require_once "Email.class.inc";
+
 $client = new NDB_Client();
 $client->initialize("../../project/config.xml");
+
 $factory = NDB_Factory::singleton();
 $baseURL = $factory->settings()->getBaseURL();
 
-$config = NDB_Config::singleton();
-
-// create Database object
-$DB =& Database::singleton();
+// Setup Database object.
+$config    =& \NDB_Config::singleton();
+$db_config = $config->getSetting('database');
+$db        = \Database::singleton(
```

Comment:
Why is there a initialization to the db class with parameters? Can't it be done just with `$db =& \Database::singleton();`?
Tested it out on my instance, and I was getting an error, and when replaced with the above, it worked fine.